src/flexi_logger.rs

```diff
@@ -37,47 +46,8 @@ impl FlexiLogger {
                                 .unwrap(/* catch and expose error? */)
                                 .enabled(level, module)
     }
-}
-
-impl log::Log for FlexiLogger {
-    //  If other writers are configured and the metadata target addresses them correctly,
-    //      - we should determine if the metadata-level is digested by any of the writers
-    //        (including the primary writer)
-    //  else we fall back to default behavior:
-    //      Return true if
-    //      - target is filled with module path and level is accepted by log specification
-    //      - target is filled with crap and ???
-    //
-    // Caveat:
-    // Rocket e.g. sets target explicitly to several fantasy names;
-    // these hopefully do not collide with any of the modules in the log specification;
-    // since they do not conform with the {} syntax expected by flexi_logger, they're treated as
-    // module names.
-    fn enabled(&self, metadata: &log::Metadata) -> bool {
-        let target = metadata.target();
-        let level = metadata.level();
-
-        if !self.other_writers.is_empty() && target.starts_with('{') {
-            // at least one other writer is configured _and_ addressed
-            let targets: Vec<&str> = target[1..(target.len() - 1)].split(',').collect();
-            for t in targets {
-                if t != "_Default" {
-                    match self.other_writers.get(t) {
-                        None => eprintln!("[flexi_logger] bad writer spec: {}", t),
-                        Some(writer) => {
-                            if level < writer.max_log_level() {
-                                return true;
-                            }
-                        }
-                    }
-                }
-            }
-        }
-
-        self.primary_enabled(level, target)
-    }
 
-    fn log(&self, record: &log::Record) {
+    fn log_impl(&self, record: &log::Record) {
```

Comment:
Would you mind moving the method log_impl before the method primary_enabled? I hope the differ will then work better :-)

Reply:
Sure, on it

Reply:
Done!, not sure the result is that good though...